petabtests/cases/v2.0.0/sbml/0016/_mapping.tsv

```diff
@@ -0,0 +1,3 @@
+petabEntityId	modelEntityId	name
+condition1		condition1
```

Comment:
Typo? Why are condition IDs in the mapping table?

Reply:
No typo. This just replaces the old `conditionName`.